requirements.txt

```diff
@@ -1,6 +0,0 @@
-# generated from manifests external_dependencies
-azure-identity
-azure-storage-blob
-prometheus_client
-python-json-logger
-redis
```

Comment:
Why removing redis, we will need it installed no ?

Reply:
Hello Vincent, you are right. That problem was because I ran pre-commit before doing the migration. I already corrected it, what do you think?